scripts/g.extension/g.extension.py

```diff
@@ -1918,7 +1918,7 @@ def resolve_xmlurl_prefix(url, source=None):
         'ignored_suffixes': ['.zip', '.tar.gz', '.tar.bz2', '.tar'],
         'possible_starts': ['', 'https://', 'http://'],
         'url_start': 'https://',
-        'url_end': '/repository/archive.zip',
+        'url_end': '/-/archive/master/{}-master.zip',
```

Comment:
I think something like `...master/{name}-master...` would be better considering that the string is far from the `.format()` call.